File: Modules/Dr-Quine/Colleen/Colleen.c
#include <stdio.h>
/*
    com1
*/
void uselessFunc(void){}
int main()
{/*
    com2
*/uselessFunc();
char*a="#include%c<stdio.h>%c/*%c%c%c%c%ccom1%c*/%cvoid%cuselessFunc(void){}%cint%cmain()%c{/*%c%c%c%c%ccom2%c*/uselessFunc();%cchar*a=%c%s%c;printf(a,32,10,10,32,32,32,32,10,10,32,10,32,10,10,32,32,32,32,10,10,34,a,34,10);}%c";printf(a,32,10,10,32,32,32,32,10,10,32,10,32,10,10,32,32,32,32,10,10,34,a,34,10);}

File: Modules/Dr-Quine/Bonus/Colleen/Colleen.rb
# com1
def uselessFunc()
   return
end
# com2
str = "# com1%cdef uselessFunc()%c   return%cend%c# com2%cstr = %c%s%c%cprintf(str,10,10,10,10,10,34,str,34,10,10)%c"
printf(str,10,10,10,10,10,34,str,34,10,10)
